collect_app/src/main/java/org/odk/collect/android/views/ODKView.java

```diff
@@ -483,7 +484,22 @@ public void highlightWidget(FormIndex formIndex) {
                 scrollTo(0, qw.getTop());
 
                 ValueAnimator va = new ValueAnimator();
-                va.setIntValues(getResources().getColor(R.color.red), getDrawingCacheBackgroundColor());
+                if (Build.VERSION.SDK_INT > Build.VERSION_CODES.KITKAT) {
+                    // only for kitkat and newer versions
```

Comment:
In this case the condition is very clear thanks to helpfully-named constants so this comment isn't necessary.